src/main/java/fi/otavanopisto/restfulptv/server/organizationservices/OrganizationServiceEntityUpdater.java

```diff
@@ -84,15 +84,27 @@ public void stopTimer() {
   }
 
   @Asynchronous
-  public void onOrganizationIdUpdateRequest(@Observes OrganizationServiceIdUpdateRequest event) {
+  public void onOrganizationServiceIdUpdateRequest(@Observes OrganizationServiceIdUpdateRequest event) {
     if (!stopped) {
       if (event.isPriority()) {
-        queue.remove(event.getId());
-        queue.add(0, event.getId());
+        prependToQueue(event.getIds());
       } else {
-        if (!queue.contains(event.getId())) {
-          queue.add(event.getId());
-        }
+        appendToQueue(event.getIds());
+      }
+    }
+  }
+
+  private void prependToQueue(List<String> ids) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "prependToQueue" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)